roles/ara_api/tasks/pre-requirements.yaml

```diff
@@ -16,46 +16,12 @@
 #  You should have received a copy of the GNU General Public License
 #  along with ARA Records Ansible. If not, see <http://www.gnu.org/licenses/>.
 
-- name: Get list of installed packages
-  package_facts:
-    manager: "auto"
-  no_log: "{{ ara_api_secure_logging }}"
-
-- name: Retrieve list of missing required packages
-  set_fact:
-    ara_api_missing_packages: "{{ ara_api_required_packages | difference(ansible_facts['packages'].keys()) }}"
-
-# Only attempt to elevate privileges if there are any missing packages
```

Comment:
I'd rather not remove the logic around detecting whether or not packages are already installed and the reason why is in that comment.

In other words, the role is designed to be executable under an unprivileged user if the necessary packages have already been installed: ara can be set up and run within a user's home directory without relying on root or sudo privileges.

By removing this logic, ansible will try to elevate privileges (``become: true``) and it will not work if Ansible isn't running with root or sudo.

We can remove ``warn: false`` if it is an issue, which leaves the fix looking like this instead: https://github.com/Thulium-Drake/ara-collection/commit/91ae432d082c045579ed4b56c35c574997695c72 (thanks @Thulium-Drake).